infra/eureka/src/main/resources/application-dev.yml

```
@@ -0,0 +1,15 @@
+
+server:
+  port: 19090
+
+eureka:
+  client:
+    register-with-eureka: false
+    fetch-registry: false
+    service-url:
+      defaultZone: ${EUREKA_SERVER_URL}
+  instance:
+    hostname: localhost
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)